tests/integration-tests/conftest.py

```diff
@@ -176,11 +176,11 @@ def _cluster_factory(cluster_config):
 def _write_cluster_config_to_outdir(request, cluster_config):
     out_dir = request.config.getoption("output_dir")
     os.makedirs(
-        "{out_dir}/clusters_configs/{test_dir}".format(out_dir=out_dir, test_dir=os.path.dirname(request.node.name)),
+        "{out_dir}/clusters_configs/{test_dir}".format(out_dir=out_dir, test_dir=os.path.dirname(request.node.nodeid)),
         exist_ok=True,
     )
     cluster_config_dst = "{out_dir}/clusters_configs/{test_name}.config".format(
-        out_dir=out_dir, test_name=request.node.name
+        out_dir=out_dir, test_name=request.node.nodeid.replace("::", "_")
```

Comment:
Shall we use a different separator since `_` is already in the test file name and in the test function name? Also `-` might work for me

Reply:
fixed